integration-tests/external-plugin-test/external-plugin-common/pom.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>external-plugin-test</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.21.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>external-plugin-common</artifactId>
```

Comment:
[question] I'm not sure about the "external-plugin-common" naming here, as it's just an internal module of the test application that we have to ship as a regular jar to allow the external plugin to have it as a dependency.

Do you think we could have something that makes it more obvious that it's a module of the application and not the external plugin ? Using a very generic `app-XXX` with `XXX=internal` for example might prevent having to explain what modules does what.

Reply:
Not sure what you mean. It is required in both other modules, not only the test app. The unit test (`PluginInstrumentationTest`) also uses it.

Reply:
Renamed to `plugin-instrumentation-target`